src/Workspaces/Core/Portable/FindSymbols/ReferenceLocation.cs

```diff
@@ -50,14 +56,14 @@ public struct ReferenceLocation : IComparable<ReferenceLocation>, IEquatable<Ref
 
         public CandidateReason CandidateReason { get; }
 
-        internal ReferenceLocation(Document document, IAliasSymbol alias, Location location, bool isImplicit, bool isWrittenTo, CandidateReason candidateReason)
+        internal ReferenceLocation(Document document, IAliasSymbol alias, Location location, bool isImplicit, ValueUsageInfo valueUsageInfo, CandidateReason candidateReason)
```

Comment:
this may be called from another language (not sure, but we need to check).  If so, we might need to leave the old constructor, and have it delegate to a new one.  

However, if this is truly C#/VB only, then this is safe. i just dont remember anymore.

Reply:
Verified this is only used within Roslyn.sln.

---
In reply to: [220754806](https://github.com/dotnet/roslyn/pull/30155#discussion_r220754806) [](ancestors = 220754806)